plugins/clipboard/plugin.js

```diff
@@ -1556,6 +1652,31 @@
 		 */
 		mainPasteEvent: ( CKEDITOR.env.ie && !CKEDITOR.env.edge ) ? 'beforepaste' : 'paste',
 
+		/**
+		 * Adds a new paste button to the editor.
+		 *
+		 * This method should be called for buttons that should display the Paste Dialog fallback in mobile environments.
+		 * See {@link https://github.com/ckeditor/ckeditor-dev/issues/595#issuecomment-345971174 the rationale} for more
+		 * details.
+		 *
+		 * @since 4.9.0
+		 * @param {CKEDITOR.editor} editor The editor instance.
+		 * @param {String} name Name of the button.
+		 * @param {Object} definition Definition of the button.
+		 */
+		addPasteButton: function( editor, name, definition ) {
```

Comment:
`addPasteButton` should return created `CKEDITOR.ui.button` instance, so that we could use it for instance with Balloon Toolbar.

Reply:
Due to the super convoluted way of accessing button instance from our UI, we decided to not introduce this change. Info gathered from `editor._.pasteButtons` should be enough to create buttons for Balloon Toolbar.